src/components/Dropdown/Dropdown.js

```diff
@@ -27,11 +27,11 @@ const Dropdown = ({ buttonText, list, onSelect, selectedItem, theme }) => {
             `}
         >
             <div className={styles.clickArea} onClick={toggleDropdown}>
-                <span>{buttonText}</span>
+                <span>Filter by Region</span>
```

Comment:
Why did you do this? If you hard code the value component cannot be reused in other places in the app, if for example we would need an other dropdown to select language of a country we would need to make an other component for that as this one is hard coded for regions. This is bad idea.

Reply:
I was not aware if we have plans to create more filter options